server/core/src/main/java/dev/slimevr/config/KeybindingsConfig.java

```diff
@@ -8,12 +8,16 @@ public class KeybindingsConfig {
 
 	private String mountingResetBinding = "CTRL+ALT+SHIFT+I";
 
+	private String pauseTrackingBinding = "CTRL+ALT+SHIFT+O";
```

Comment:
this should totally be P for Pause instead of O

Reply:
I just followed the row of keys :P